src/main/resources/config/liquibase/changelog/20210624110700_changelog.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="schlesinger" id="20210603110700">
+
+        <!-- Introduction of post tags -->
+        <createTable tableName="post_tag">
+            <column name="post_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="text" type="VARCHAR(50)"/>
+        </createTable>
+        <createIndex indexName="fk_post_tag_post_id" tableName="post_tag">
+            <column name="post_id"/>
+        </createIndex>
+        <addForeignKeyConstraint baseColumnNames="post_id" baseTableName="post_tag" constraintName="fk_post_tag_post_id" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="post"/>
+
+        <!-- Introduction of course-wide context besides lecture and exercise link -->
+        <addColumn tableName="post">
+            <column name="course_id" type="BIGINT"/>
+        </addColumn>
+        <addForeignKeyConstraint baseColumnNames="course_id" baseTableName="post" constraintName="fk_post_course_id" deferrable="false" initiallyDeferred="false" referencedColumnNames="id" referencedTableName="course"/>
+        <addColumn tableName="post">
+            <column name="course_wide_context" type="VARCHAR(50)"/>
+        </addColumn>
+
+        <!-- Introduction of reactions on postings, to be represented by emojis -->
+        <createTable tableName="reaction">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="reactionPK"/>
+            </column>
+            <column name="emoji_id" type="VARCHAR(20)">
```

Comment:
like above in the java code, this could either be a BIGINT too, or changed to emoji_name.

Reply:
see comment https://github.com/ls1intum/Artemis/pull/3527/files#r660472952